qa/tasks/cephfs/test_nfs.py

```diff
@@ -1,12 +1,8 @@
-import os
```

Comment:
@tchaikov Can you please create a separate commit for  `qa/tasks/cephfs/test_nfs.py` ? It will be helpful in backporting of nfs-ganesha related prs to octopus.  https://github.com/ceph/ceph/pull/35499

Reply:
i can. but this change is considered a cleanup. why do you want to backport it?

Reply:
I have other nfs-ganesha prs(https://github.com/ceph/ceph/pull/35417, https://github.com/ceph/ceph/pull/35646) based on master which I will be backporting and will probably be rebased on your pr. If it gets merged first. To avoid conflicts I want to do backport.

Reply:
@varshar16 extracted into a separate commit.

Reply:
@tchaikov Thank you.